mac/Info.plist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>BeeRef</string>
+	<key>CFBundleExecutable</key>
+	<string>BeeRef</string>
+	<key>CFBundleIconFile</key>
+	<string>logo.icns</string>
+	<key>CFBundleIdentifier</key>
+	<string>BeeRef</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>BeeRef</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>0.1.1</string>
+	<key>NSHighResolutionCapable</key>
+	<true/>
+	<key>CFBundleDocumentTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeExtensions</key>
+			<array>
+				<string>bee</string>
+			</array>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+		</dict>
+	</array>
+</dict>
+</plist>
```

Comment:
This file is the biggest problem. I currently manually replace `Info.plist` generated by `pyinstaller`  with this file. The bit relevant to this PR is the `CFBundleDocumentTypes` section.
I was not sure how to properly include this step in the build processes.

Reply:
Pyinstaller has an option for adding plist values. It only works when the program is bundled into one single executable instead of the current zipped folder. I do want to have a single executable anyway, but so far PyInstaller didn't support PyQt6 properly yet — however, I see there's been a new update recently that promises PyQt6 compatibility. Let me try it out, then I'll get back to you.

Reply:
Okay, I've made a couple of changes to how pyinstaller is called. Please rebase your branch. Then you'll see that there's now a file called BeeRef.spec. At the end of that file, you'll find a Mac-specific section. That's the place where you can add plist options as per these instructions:

https://pyinstaller.readthedocs.io/en/stable/spec-files.html#spec-file-options-for-a-mac-os-x-bundle

You can then build by running `pyinstaller --onefile BeeRef.spec`

Let me know if you have any more questions, or if something doesn't work. It's not too thoroughly tested yet.